misk-testing/src/main/kotlin/misk/testing/LogLevelExtension.kt

```diff
@@ -0,0 +1,46 @@
+package misk.testing
+
+import ch.qos.logback.classic.Level
+import ch.qos.logback.classic.Logger
+import org.junit.jupiter.api.extension.BeforeEachCallback
+import org.junit.jupiter.api.extension.ExtensionContext
+import org.slf4j.LoggerFactory
+import javax.inject.Inject
+
+/**
+ * Helps you set the log level for misk tests. It would default all test by default to Log
+ * but you can override it by using the [LogLevel] annotation at the method or class level.
+ *
+ */
+class LogLevelExtension @Inject constructor() : BeforeEachCallback {
+  override fun beforeEach(context: ExtensionContext?) {
+    val root = LoggerFactory.getLogger(Logger.ROOT_LOGGER_NAME) as Logger
+    root.level =  level(context)
+  }
+
+  private fun level(context: ExtensionContext?): Level {
+    if(context == null || context.element.isEmpty) {
+      return Level.INFO
+    }
+    return  context.element.get().annotations.firstNotNullOfOrNull { it as? LogLevel }
+      ?.let { mapLevel(it.level) }
+      ?: findParent(context)
+      ?: Level.INFO
+  }
+
+  private fun findParent(context: ExtensionContext): Level? {
```

Comment:
This method lets you annotate the test class so all the tests within that class (including nested tests) will use that log level. It might be a bit of an overkill, and I'm open to removing it.